givre/src/signing/aggregate.rs

```diff
@@ -86,6 +86,14 @@ impl<C: Ciphersuite> Signature<C> {
     }
 }
 
+impl<C: Ciphersuite> PartialEq for Signature<C> {
+    fn eq(&self, other: &Self) -> bool {
+        let Signature { r, z } = self;
+        *r == other.r && *z == other.z
+    }
+}
+impl<C: Ciphersuite> Eq for Signature<C> {}
```

Comment:
Why not derive? Because of Ciphersuite not being Eq? Can we make it?

Reply:
Yea, you're right, I think it's better to add `Eq` to the `Ciphersuite`